package.json

```diff
@@ -141,6 +148,9 @@
     "typescript": "3.5.3",
     "webpack": "4.39.2"
   },
+  "bin": {
+    "instantsearch-scripts": "./scripts/index.js"
```

Comment:
Why not have this index file in `scripts/transforms/bin.js` and only include this folder in `files`?

Reply:
It’s possible, I was thinking of potentially adding more cli later, but I guess that’s premature Optimization